.github/workflows/npm-cd.yml

```diff
@@ -284,7 +257,7 @@ jobs:
                   NODE_AUTH_TOKEN: ${{ secrets.NPM_AUTH_TOKEN }}
 
     test-release-candidate:
```

Comment:
why are we testing only the release candidate? and if it isn't the case - should be renamed

Reply:
It is an old name. Will rename.